Public/Invoke-DockerPush.ps1

```diff
@@ -3,19 +3,26 @@
 function Invoke-DockerPush {
     [CmdletBinding()]
     param (
+        [Parameter(ValueFromPipelineByPropertyName = $true)]
         [ValidateNotNull()]
         [string]
         $Registry = '',
 
-        [Parameter(Mandatory = $true)]
+        [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
         [string]
         $ImageName,
 
+        [Parameter(ValueFromPipelineByPropertyName = $true)]
         [ValidateNotNullOrEmpty()]
         [string]
         $Tag = 'latest'
     )
     $postfixedRegistry = Add-PostFix $Registry
     $command = "docker push ${postfixedRegistry}${ImageName}:${Tag}"
-    Invoke-Command $command
+    $commandResult = Invoke-Command $command
+    [PSCustomObject]@{
```

Comment:
And now no result object just returning the object 😰 

I feel like we should choose one style. 😄